index.html

```diff
@@ -234,10 +235,10 @@
       <li>Working with vectors and data frames</li>
       <li>Reading and plotting data</li>
       <li>Creating and using functions</li>
-      <li>Loops and conditionals</li>
-      <li>Using R from the command line</li>
```

Comment:
I still don't see the point of removing this. These are valid R topics.  Whether we cover them or not.